### DISCUSSION TITLE
多数据库连接支持

### DISCUSSION THREAD

**Discussion by fzwu** [luoyunchong/lin-cms-dotnetcore#34] [Q&A]
由于业务需要，应用程序需要根据接口访问不同的数据库，如何根据请求接口来访问不同数据库

**Comment by luoyunchong** [comment]
https://freesql.net/guide/sharding.html#%E3%80%90%E5%88%86%E5%BA%93%E3%80%91%E4%BD%BF%E7%94%A8-freesql-cloud
